bingosync-app/static/bingosync/style.css

```diff
@@ -84,6 +92,10 @@
     overflow-y: scroll;
 }
 
+.panel-body {
+    padding:8px;
```

Comment:
I definitely do prefer it with more padding. 

Overriding to 13px, specifically overriding the Room Settings panel, and changing the text size in the body of the "settings" panels all work for me as solutions. Feel free to do whichever you prefer.